src/scenic/core/regions.py

```diff
@@ -886,7 +886,11 @@ def sampleGiven(self, value):
             dimensions=value[self.dimensions],
             position=value[self.position],
             rotation=value[self.rotation],
-            orientation=value[self.orientation],
+            orientation=(
+                True
+                if self.__dict__.get("_usingDefaultOrientation", False)
+                else value[self.orientation]
+            ),
```

Comment:
What's wrong with just `value[self.orientation]` here? If `self.orientation` is `True` it will get passed through `value` unchanged.

Reply:
Hmm, but wouldn't it get converted to the default vector field on line 1790?

Reply:
I get it now -- never mind!